src/create-client.js

```diff
@@ -35,7 +35,7 @@ export const clientConstructor = ({ token, ...options }) => {
         }
       })
         .then(response => response.json())
-        .catch(error => { throw new Error(error) })
+        .catch(error => { throw new Error(error) });
```

Comment:
If we're not doing anything here other than re-throwing, we could just skip catching here.